server/chat/background/task.py

```
@@ -1919,6 +1948,10 @@ def cleanup_stale_background_chats() -> Dict[str, Any]:
                 cleaned_count = cursor.rowcount
             conn.commit()
             
+            # Propagate failed status to any linked suggestions
+            for session_id, user_id, incident_id in stale_sessions:
```

Ruff (0.15.9)

[warning] Loop control variable `user_id` not used within loop body

(B007)